triggers/incident-triggered/event.schema.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "incident": {
+      "type": "string",
+      "description": "The incident's identifier"
+    },
+    "webhook_payload": {
```

Comment:
I think this should just be the top-level event, and not a separate subkey. If we need to break it out in the future we can rev the trigger version.

Reply:
I will make this change, but have some trepidation because the fields that are inside it aren't documented or reliably available